code-organization.md

```diff
@@ -12,17 +12,23 @@ TL;DR
 For those of you looking for a little more from this page,
 please keep reading.
 
-## File Organization
+## Static Files Organization
 
 JavaScript can currently live in a few different places.
-Here are the main places it tends to show up in the file structure
-of a Django app:
+JavaScript files belong in the `static` directory of a Django app,
+which we structure as follows:
 
 ```
 myapp/
   static/myapp/
-    js/
-      myjsfile.js
+    css/
+    font/
+    images/
+    js/       <= JavaScript
+    less/
+    lib/      <= Third-party code
```

Comment:
feel like this should never be needed and we should always lean towards bower, but guessing there are still a few instances where we use lib instead of bower